### ISSUE TITLE
Add OverflowTemplate to span overflow control on traces page

### ISSUE THREAD

**Issue by JamesNK** [dotnet/aspire#2214]
Today overflowed spans are displayed as a list of text. We can use OverflowTemplate to show spans in the tooltip the same way they're shown in the grid (i.e. color + border).

Low priority.

Note: Blocked on FluentOverflow control being fixed by updating FluentUI to 4.4.x +.

**Comment by JamesNK** [comment]
I did this at some point.

![image](https://github.com/user-attachments/assets/00282e0f-6d36-4f57-8b8d-f10cd802a3f5)
